doc/reference/configuration/configuration_reference.rst

```diff
@@ -1680,6 +1680,135 @@ The ``groups`` section provides the ability to define the :ref:`full topology of
     For example, :ref:`iproto <configuration_reference_iproto>` and :ref:`database <configuration_reference_database>` configuration parameters defined at the group level are applied to all instances in this group.
 
 
+..  _configuration_reference_memtx:
+
+memtx
+-----
+
+The ``memtx`` section is used to configure parameters related to :ref:`memtx engine <configuration_memtx>`.
+
+.. NOTE::
+
+    ``memtx`` can be defined in any :ref:`scope <configuration_scopes>`.
+
+
+-   :ref:`memtx.allocator <configuration_reference_memtx_allocator>`
+-   :ref:`memtx.max_tuple_size <configuration_reference_memtx_max_size>`
+-   :ref:`memtx.memory <configuration_reference_memtx_memory>`
+-   :ref:`memtx.min_tuple_size <configuration_reference_memtx_min_size>`
+-   :ref:`memtx.slab_alloc_factor <configuration_reference_memtx_slab_alloc_factor>`
+-   :ref:`memtx.slab_alloc_granularity <configuration_reference_memtx_slab_alloc_granularity>`
+-   :ref:`memtx.sort_threads <configuration_reference_memtx_sort_threads>`
+
+..  _configuration_reference_memtx_allocator:
+
+..  confval:: memtx.allocator
+
+    Specify the allocator used for memtx tuples.
+    The possible values are ``system``  and ``small``:
+
+    *   ``system`` is based on the ``malloc`` function.
+        The allocator allocates memory as needed, checking that the quota is not exceeded.
+
+    *   ``small`` is a special `slab allocator <https://github.com/tarantool/small>`_.
+        The allocator repeatedly uses a memory block to allocate objects of the same type.
+        Note that this allocator is prone to unresolvable fragmentation on specific workloads,
+        so you can switch to ``system`` in such cases.
+
+    |
+    | Type: string
+    | Default: 'small'
+    | Environment variable: TT_MEMTX_ALLOCATOR
+
+..  _configuration_reference_memtx_max_size:
+
+..  confval:: memtx.max_tuple_size
+
+    Size of the largest allocation unit for the memtx storage engine in bytes.
+    It can be increased if it is necessary to store large tuples.
+    See also: ``vinyl.max_tuple_size``.
```

Comment:
Let's drop links to vinyl options. Looks like they are not relevant for memtx users.

```suggestion
    It can be increased if it is necessary to store large tuples.
```